homeassistant/components/light/tplink.py

```diff
@@ -141,16 +152,12 @@ def update(self):
         """Update the TP-Link Bulb's state."""
         from pyHS100 import SmartDeviceException
         try:
-            if self._supported_features == 0:
+            if not self._supported_features:
```

Comment:
You need to check for `is None`, `not` will still think that a 0 means it needs to fetch features.

Reply:
Indeed, fixed.